assets/js/googlesitekit/data/index.js

```diff
@@ -52,15 +43,6 @@ const Data = createRegistry();
 
 // Attach some of our utility functions to the registry so third-party
 // developers can use them.
-Data.addInitializeAction = addInitializeAction;
-Data.addInitializeReducer = addInitializeReducer;
-Data.collectActions = collectActions;
-Data.collectControls = collectControls;
-Data.collectReducers = collectReducers;
-Data.collectResolvers = collectResolvers;
-Data.collectSelectors = collectSelectors;
-Data.collectState = collectState;
-Data.collectName = collectName;
```

Comment:
Shouldn't these changes be part of #1484 instead?

Reply:
Ah yeah, sorry about that, I inadvertently merged them in when troubleshooting a build issue. I'll remove the extra changes.